content/en/agent/kubernetes/_index.md

```diff
@@ -41,6 +41,8 @@ Run the Datadog Agent in your Kubernetes cluster as a DaemonSet in order to star
 
 ## Installation
 
+**Note**: We have dedicated documentation and examples for [all major Kubernetes distributions][15] (GKE, EKS, AKS, OpenShift, Rancher, etc.)
```

Comment:
Noticed that we didn't add Rancher yet. Are we planning to in this PR? Otherwise it's okay to remove it from this line

Reply:
Actually taking a step back here. We should only defer users to this page if they have an environment like AKS that requires special configs. Therefore, let's move this note after the default instructions and say something like:

The following Kubernetes environments require additional configurations:
1. Azure Kubernetes Service (AKS)
2. Red Hat OpenShift

and each should be a link to the right section in the other page

Reply:
Agree with Vincent. If a user doesn't find GKE in this list, he might think that we don't support it.